applications/RANSApplication/custom_python/rans_python_application.cpp

```diff
@@ -97,6 +97,11 @@ PYBIND11_MODULE(KratosRANSApplication,m)
     KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, WALL_SMOOTHNESS_BETA )
     KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, RANS_IS_WALL_FUNCTION_ACTIVE )
     KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, FRICTION_VELOCITY )
+
+    // formulation specific variables
+    KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, WALL_MODEL_PART_NAME )
```

Comment:
It is not used in here. Its used in coming PRs.

Reply:
No. All the slip boundaries need to be unified in a single model part prior to the nodal normal calculation to avoid issues with the nodal normal calculation in the corners (or in those nodes in where the union between modelparts occurs). This is the unique way you can ensure that the corner normals are properly computed.

This can be done when writting the mpda (what we do right now) or afterwards with a sort of utility like the one you propose in #7382. The point is that we need to somehow provide this information.